common/src/test/java/com/graphicsfuzz/common/util/FragmentShaderToShaderJobTest.java

```diff
@@ -90,6 +90,7 @@ public void basicJsonGenerationTest() throws Exception {
     assertTrue(vertdata.contains("out vec2 in_v2;"));
     assertTrue(vertdata.contains("out vec3 in_v3;"));
     assertTrue(vertdata.contains("out vec4 in_v4;"));
+    assertTrue(vertdata.contains("gl_Position = _GLF_pos;"));
```

Comment:
And please use Constants.GLF_POS here once you have added it.

Reply:
Done

Reply:
I had meant:

assertTrue(vertdata.contains("gl_Position = " + Constants.GLF_POS + ";"));

just in case we change our mind about the name of this variable later.